src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard.ts

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react'
+import { h } from 'react-hyperscript-helpers'
+import { spinnerOverlay } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import { useLoadedData } from 'src/libs/ajax/loaded-data/useLoadedData'
+import { useCancellation } from 'src/libs/react-utils'
+import { AzureManagedAppCoordinates } from 'src/pages/billing/models/AzureManagedAppCoordinates'
+import { BillingProject } from 'src/pages/billing/models/BillingProject'
+import { StepWizard } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepWizard'
+
+import { AddUserStep } from './AddUserStep'
+import { AzureSubscriptionIdStep } from './AzureSubscriptionIdStep'
+import { CreateProjectStep } from './CreateProjectStep'
+
+
+interface AzureBillingProjectWizardProps {
+
+}
+
+export const AzureBillingProjectWizard = ({ ...props }: AzureBillingProjectWizardProps) => {
```

Comment:
If I am understanding the pieces of this PR correctly, GCPBillingProjectWizard is meant to replace the existing wizard mechanics.... and AzureBillingProjectWizard does not appear to be reference by anything yet...

I recommend this PR gets split apart to wait on the AzureBillingProjectWizard bits until they can be paired with unit tests in their own PR.
- reduce the size of this PR for easier review
- avoid code-coverage stats going down because of new un-covered code going in without tests.

Reply:
I can split out the changes without the azure part, now that I know how the components need to work.